packages/docs/package.json

```diff
@@ -76,7 +76,7 @@
     "codesandbox.sync": "tsx codesandbox.sync.ts",
     "contributors": "tsx contributors.ts",
     "deploy": "wrangler pages publish ./dist",
-    "dev": "tsx check-qwik-build.ts && vite --mode ssr --open",
+    "dev": "concurrently \"tsx check-qwik-build.ts\" \"vite --mode ssr --open\"",
```

Comment:
This one is riskier, it is a race condition. The build check isn't that slow is it?

Reply:
Yeah I understood that, both the cammands are getting executed cucurrently but, this cammand :- 
"dev": "tsx check-qwik-build.ts && vite --mode ssr --open"
is not working on my both windows machines and it was recommended by @gioboa as it was causing issues in running the project locally.
If needed it can be deleted if, it works !

Reply:
&& should work on windows too. Perhaps it broke for a different reason?

Reply:
Should I make it && then?

Reply:
You should make it what it was before and investigate why it didn't work on windows

Reply:
@wmertens
Will try and let you know.